.github/workflows/release-code-freeze.yml

```diff
@@ -6,6 +6,10 @@ on:
 
 permissions:
   contents: write
+  pull-requests: write
```

Comment:
I was getting:
```
Invalid workflow file: .github/workflows/release-code-freeze.yml#L146
The workflow is not valid. .github/workflows/release-code-freeze.yml (Line: 146, Col: 3): Error calling workflow 'albarin/woocommerce/.github/workflows/release-bump-version.yml@9e17de9048cbf6281e5817091b7b1004fef7b7b8'.
The workflow is requesting 'issues: write', but is only allowed 'issues: none'.
```

I think the `issues: write` permission is also needed in this workflow.

Reply:
That's what I get for [trusting the rabbit](https://github.com/woocommerce/woocommerce/pull/59946#discussion_r2226413148). Fixed in bf63efe917d482eb7b06b67ff4717c0fa1c98323.